From: Chris Hanson <cph@debian.org>
Date: Wed, 2 Dec 2009 11:22:04 +0000
Subject: char mhash_get_hash_name

Change return type in prototype of mhash_get_hash_name from
mutils_word8* to char*.  See http://bugs.debian.org/354206
---
 include/mutils/mhash.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/mutils/mhash.h b/include/mutils/mhash.h
index 469c149..fbddc45 100644
--- a/include/mutils/mhash.h
+++ b/include/mutils/mhash.h
@@ -55,7 +55,7 @@ typedef struct mhash_hash_entry mhash_hash_entry;
 
 mutils_word32 mhash_count(void);
 mutils_word32 mhash_get_block_size(hashid type);
-mutils_word8 *mhash_get_hash_name(hashid type);
+char *mhash_get_hash_name(hashid type);
 void mhash_free(void *ptr);
 
 __const mutils_word8 *mhash_get_hash_name_static(hashid type);
